code/game/machinery/computer/card.dm

```diff
@@ -624,7 +457,41 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 
 	light_color = LIGHT_COLOR_ORANGE
 
-#undef JOB_ALLOWED
-#undef JOB_COOLDOWN
-#undef JOB_MAX_POSITIONS
-#undef JOB_DENIED
+/obj/machinery/computer/card/ncr
+	name = "\improper NCR identification console"
+	circuit = /obj/item/circuitboard/computer/card/ncr
+	// The job list is here so only specific titles are allowed.
+	job_list = list(
+		"NCR Citizen",
+		"NCR Trooper",
+		"NCR Corporal",
+		"NCR Combat Medic",
+		"NCR Combat Engineer",
+		"NCR Sergeant",
+	)
+
+/obj/machinery/computer/card/ncr/ui_interact(mob/user)
+	var/list/job_req = list("NCR Colonel", "NCR Captain", "NCR Lieutenant")
```

Comment:
Can you put this as an object variable, so it can be edited and redefined?

Reply:
Fair. Will do just that.